tests/unittest/_torch/ray_orchestrator/multi_gpu/test_llm_update_weights_multi_gpu.py

```
@@ -579,3 +584,166 @@ def filter_fn(name: str) -> bool:
+
+    pkgs = ["mamba-ssm", "causal-conv1d"]
+    mod_names = {"mamba-ssm": "mamba_ssm", "causal-conv1d": "causal_conv1d"}
+    need_install = [p for p in pkgs if importlib.util.find_spec(mod_names[p]) is None]
+
+    before = _freeze() if need_install else None
+    try:
+        if need_install:
+            # --no-deps: avoid pulling in optional kernel deps (quack-kernels,
+            # tilelang) that upgrade nvidia-cutlass-dsl and break tensorrt-llm.
+            # The container already provides torch/einops/etc.
+            subprocess.check_call(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)